Directory.Packages.props

```diff
@@ -12,5 +12,6 @@
     <PackageVersion Include="Moq" Version="4.14.3" />
     <PackageVersion Include="xunit" Version="2.4.0" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageVersion Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.2.2" />
```

Comment:
Any reason why we're not using 1.2.3? https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator/

Reply:
we ran into issues with the older version when setting up the functions earlier and updating to latest solved it.

Reply:
Yeah, but latest 1.x version is 1.2.3 but we're using 1.2.2

Reply:
will update it then